DataHub/Integration/01_ManageAttributes.md

```diff
@@ -0,0 +1,219 @@
+
+#### Procedure
+
+*DataHub > Settings > Tab ATTRIBUTES*
+
+![Attributes](../../Assets/Screenshots/DataHub/Settings/Attributes/Attributes.png "[Attributes]")
+
+1. Select the checkbox of the attribute you want to delete in the list of attributes.   
+  The editing toolbar is displayed above the attributes list.
+
+2. Click the ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete) button in the toolbar. The *Confirm deletion of attribute?* window is displayed.
+
+  > [Info] In the window all dependencies to attribute sets, ETL-mappings and entity entries are displayed. Be aware that problems may occur if you delete an attribute with existing dependencies.
+
+3. Click the [CONFIRM] button in the bottom right corner.   
+  The attribute has been deleted and is moved to the *DELETED ATTRIBUTES* tab in the *DataHub* module. The *Confirm deletion of attribute?* window is closed.
+
+> [Info] The attribute is moved to the *DELETED ATTRIBUTES* tab, but it can still be recovered. To irretrievably delete the attribute, see [Finally delete an attribute](#finally-delete-an-attribute).
+
+
+
+## Recover an attribute
+
+If you have moved an attribute to the *DELETED ATTRIBUTES* tab, you can recover the attribute and make the deletion undone.
+
+#### Prerequisites
+
+At least one attribute has been moved to the *DELETED ATTRIBUTES* tab, see [Delete an attribute](#delete-an-attribute).
+
+#### Procedure
+
+*DataHub > Settings > Tab DELETED ATTRIBUTES*
+
+![Attributes](../../Assets/Screenshots/DataHub/Settings/DeletedAttributes/DeletedAttributes.png "[Attributes]")
+
+1. Select the checkbox of the attribute you want to recover in the list of deleted attributes.   
+  The editing toolbar is displayed above the deleted attributes list.
+
+2. Click the [RECOVER] button in the toolbar.   
+  The selected attribute has been recovered and is removed from the list of deleted attributes.
+
+[comment]: <> (Why can I not recover certain attributes? Is it a bug?)
+
+
+
+## Finally delete an attribute
+
+You can finally delete an attribute if it is no longer needed.
+As there are usually dependencies on an attribute, for example through an attribute set or created products, it is strongly recommended not to delete an attribute irretrievably.
+Finally deleted attributes cannot be recovered.
+
+#### Prerequisites
+
+At least one attribute has been moved to the *DELETED ATTRIBUTES* tab, see [Delete an attribute](#delete-an-attribute).
+
+#### Procedure
+
+*DataHub > Settings > Tab ATTRIBUTES*
+
+![Attributes](../../Assets/Screenshots/DataHub/Settings/Attributes/Attributes.png "[Attributes]")
+
+1. Select the checkbox of the attribute you want to finally delete in the list of deleted attributes.    
+  The editing toolbar is displayed above the attributes list.
+
+  > [Info] If the deleted attribute is not yet displayed in the list of deleted attributes, click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button to update the list.
```

Comment:
Vorschlag: temporarily deleted oder moved to DELETED ATTRIBUTES? Sonst klingt verwirrend: If the **deleted** attribute is not yet displayed in the list of **deleted** attributes, ...